source/install/self-managed-changelog.md

```diff
@@ -5,12 +5,121 @@
+
+#### Integrations
+ - To keep users in Mattermost when opening documentation links from the **System Console > Plugin** settings page, all the links now open in another tab.
+ - Changed **Actions** post menu hover text to **Message Actions**.
+ - Updated Apps Framework to version 1.1.0 to add improved logging.
+
+#### Administration
+ - Timestamps are now enabled in the default audit configuration.
+ - The Collapsed Reply Threads configuration option was moved in the **System Console** from **Experimental** to **Site Configuration > Posts**.
+ 
+#### Enterprise Subscription
+ - Upgraded the minor version of the ElasticSearch development Docker image.
+ - The Support Packet now contains two additional fields in the ``support_packet.yaml`` file: Active users and License-supported users.
+
+### Bug Fixes
+ - Fixed an issue with ADA Accessibility where screen readers did not TAB to or read "This channel has guests" in the channel header bar.
+ - Fixed an issue where the @mention autosuggest of users was no longer grouped by channel membership status.
+ - Fixed an issue where the New Messages toast was not fully tappable in narrow view.
+ - Fixed an issue where the shortcut modal for channel info showed ``ALT`` instead of ``SHIFT`` for Mac.
+ - Fixed an issue where the **Help > Report a Problem** link was not hidden when a URL was not set for **System Console > Customization > Report a Problem**.
+ - Fixed an issue with the timing of selector performance metrics.
+ - Fixed an issue where the S3 **Test Connection** button deceptively failed unless the user pressed **Save** first.
+ - Fixed an issue where Workspace Optimization did not load in the System Console on subpath servers.
+ - Fixed an issue where an error was logged when ``SendEmailNotifications`` was not set to ``true``.
+   
+### config.json
+Multiple setting options were added to ``config.json``. Below is a list of the additions and their default values on install. The settings can be modified in ``config.json``, or the System Console when available.
+
+#### Changes to Team Edition and Enterprise Edition:
+ - Under ``ServiceSettings`` in ``config.json``:
+   - Changed ``SessionLengthWebInDays`` to ``SessionLengthWebInHours``.
+   - Changed ``SessionLengthMobileInDays`` to ``SessionLengthMobileInHours``.
+   - Changed ``SessionLengthSSOInDays`` to ``SessionLengthSSOInHours``.
+   - The value of ``TrustedProxyIPHeader`` defaults to empty from now on. A previous bug prevented this from happening in certain conditions. Customers are requested to check for these values in their config and set them to nil if necessary.
+   - Added ``always-on`` and ``default-on`` settings to **System Console > Experimental Features** for Collapsed Reply Threads. When enabled (default-on), users see Collapsed Reply Threads by default and have the option to disable it in **Settings**. When always on, users are required to use Collapsed Reply Threads and can't disable it.
```

Comment:
Should this path point to the new location of these settings?  "System Console > Posts" as mentioned above?